src/format/mod.rs

```diff
@@ -357,6 +357,7 @@ impl ParseError {
 
 /// The category of parse error
 #[derive(Debug, Clone, PartialEq, Eq, Copy, Hash)]
+#[non_exhaustive]
```

Comment:
ParseErrorKind was already converted as part of https://github.com/chronotope/chrono/pull/901.